db/migrate/20240108184340_create_posts.rb

```diff
@@ -5,7 +5,7 @@ def change
       t.string :text
       t.integer :likes_counter
       t.integer :comments_counter
-
+      
       t.timestamps
```

Comment:
- [ ] The posts table should contain a reference to the users table, through an `author_id` column.

  ```
  t.references :author, foreign_key: { to_table: 'users' }
  ```